src/Services/UrlBuilder/InternalUrlBuilder.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace IO\Services\UrlBuilder;
+
+use IO\Extensions\Constants\ShopUrls;
+use Plenty\Modules\Authorization\Services\AuthHelper;
+use Plenty\Modules\Frontend\Events\FrontendLanguageChanged;
+use Plenty\Modules\Order\Contracts\OrderRepositoryContract;
+use IO\Helper\MemoryCache;
+use Plenty\Plugin\Events\Dispatcher;
+
+class InternalUrlBuilder
```

Comment:
Die Klasse ist überflüssig. Die Logik wäre in der Klasse `ShopURLs` besser aufgehoben.